reborn.py

```diff
@@ -162,9 +212,39 @@ def utcToLocalizedTzs(inputTime):
 	if scheduled_utc_time < time_now:
 		scheduled_utc_time + timedelta(days=1)
 
+	return scheduled_utc_time
+
+def utcToLocalizedTzs(inputTime):
+	scheduled_utc_time = generateTimeObject(inputTime)
+	
 	localizedTimes = [
 		('• ' + scheduled_utc_time.astimezone(pytz.timezone(tz)).strftime('%H:%M') + ' ' + tz) for tz in timezones
 	]
 	return '**Localized Timezones:** \n' + "\n".join(localizedTimes) + '\n\n'
 
-bot.run(botToken)
\ No newline at end of file
+async def checkEvents():
+	await bot.wait_until_ready()
+	while True:
+		#print('checking events',flush=True)
+		time_now = datetime.now(tz=pytz.utc)
+		for event, list in eventTracker.items():
+			temp_list=[]
+			if list:
+				for t in list:
+					#print(t, flush=True)
+					if t - timedelta(minutes=15) < time_now:
+						temp_list.append(t)
```

Comment:
can we not just remove it here to avoid a loop at 239?

Reply:
I think it could cause some problems, since we'd be removing it from the list while still iterating over it. I think it could work if we created a copy of the list before the loop and iterated through that instead. That'd probably be better

Reply:
Oh nvm I tested removing it there as is and it worked, so I guess it's not a problem